src/controllers/verifyCommand.ts

```diff
@@ -13,7 +13,7 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
-
+  
```

Comment:
Nit : If in this file no changes are there this can be removed

Reply:
done